scripts/lint.py

```diff
@@ -41,9 +41,6 @@ def lint_line(line: str) -> Optional[str]:
     if "todo!()" in line:
         return 'todo!() should be written as todo!("$details")'
 
-    if "dbg!(" in line and not line.startswith("//"):
```

Comment:
Any particular reason we're removing this?

Reply:
It is (better) checked by `clippy` in `clippy.toml`